docs/architecture/microservices/microservice-ddd-cqrs-patterns/implement-value-objects.md

```diff
@@ -109,12 +125,28 @@ public class Address : ValueObject
 }
 ```
 
-You can see how this value object implementation of Address has no identity and therefore, no ID field, neither at the Address class not even at the ValueObject class.
+You can see how this value object implementation of `Address` has no identity and therefore, no ID field, neither at the `Address` class definition nor the `ValueObject` class definition.
```

Comment:
Acrolinx advises to avoid neither/nor

```suggestion
This value object implementation of `Address` has no identity, and therefore no ID field is defined for it, either in the `Address` class definition or the `ValueObject` class definition.
```